pkg/datasource/csvsource/bybit_tick_to_kline.go

```diff
@@ -0,0 +1,197 @@
+package csvsource
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+var klines []types.KLine
+
+type BybitCsvTick struct {
+	Timestamp       int64            `json:"timestamp"`
+	Symbol          string           `json:"symbol"`
+	Side            string           `json:"side"`
+	TickDirection   string           `json:"tickDirection"`
+	Size            fixedpoint.Value `json:"size"`
+	Price           fixedpoint.Value `json:"price"`
+	HomeNotional    fixedpoint.Value `json:"homeNotional"`
+	ForeignNotional fixedpoint.Value `json:"foreignNotional"`
+}
+
+func ConvertTicksToKLines(symbol string, interval time.Duration) error {
+	err := filepath.Walk(
+		fmt.Sprintf("pkg/datasource/csv/testdata/bybit/%s/", symbol),
```

Comment:
do you want to put the test data in pkg/datasource/csvsource directory? the path looks like different

Reply:
good call